server/api/impl/admin/provision.go

```diff
@@ -45,6 +45,7 @@ func (adm *Admin) ProvisionApp(cc *apps.Context, sessionToken apps.SessionToken,
 	if err != nil {
 		return nil, "", err
 	}
+	adm.store.Manifest().Save(manifest)
```

Comment:
It not obvious for the caller that the manifest also has be be saved in the store. `App().Save` should handle that.

Reply:
You are correct, but right now appStore and manifestStore are handled differently and this change will break [this PR](https://github.com/mattermost/mattermost-plugin-apps/pull/78) I'll refactor after both PRs are merged

Reply:
Could you please file a ticket for that so don't forget that?